### ISSUE TITLE
Notification setting

### ISSUE THREAD

**Issue by ElizabethRMcneill** [annathehybrid/Github#8]
https://github.com/settings/notifications![IMG_20220303_234246315.jpg](https://user-images.githubusercontent.com/100013681/157562406-5a0d39cf-9163-47eb-86e0-4e60cca5b8ec.jpg)![IMG_20220303_233357209.jpg](https://user-images.githubusercontent.com/100013681/157562443-53903766-3d19-40f4-8363-486efa46d803.jpg)****,Still fighting hacker's I am so tired of fighting to outwit men,for their female lover's wanting to be ME!!

**Comment by ElizabethRMcneill** [comment]
Yes,I will take a better photo to share very soon.KEEP IN TOUCH PLEASE!

**Comment by tristan-zander** [comment]
I am in your walls
